R/test-elements.R

```diff
@@ -1093,7 +1093,7 @@ checkbox_page <-
     )
     get_answer <- function(input, ...)
       if (is.null(input[[label]])) {
-        character(0)
+        ""
```

Comment:
Are you sure this is what we want?

Reply:
I think yes, as this caused an error in some circumstances, if I remember well. Unfortunately, I cannot recall the details as this was added quite some time ago now.

Reply:
Hmm, ok. I think I prefer the original behaviour -- no responses should yield an empty vector, not a vector with an empty string.